use-timescale/data-tiering/querying-tiered-data.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Querying Tiered Data
+excerpt: How to query tiered data
+product: [ cloud ]
+keywords: [ data tiering, tiering ]
+tags: [ storage, data management ]
+---
+
+# Querying tiered data
+
+<!-- vale Google.Acronyms = NO -->
+
+Once rarely used data is tiered and migrated to low cost object storage it can still be queried with standard SQL by
+enabling the `timescaledb.enable_tiered_reads` GUC. 
+
+The `timescaledb.enable_tiered_reads` GUC, or Grand Unified Configuration variable, is a setting that controls if tiered data is queried. 
+The configuration variable can be set at different levels, including globally for the entire database server, for individual databases, and for individual sessions.
+
+<!-- vale Google.Acronyms = YES -->
+
+<Highlight type="warning">
+By default tiered data is not queried. Querying tiered data may slow down query performance as the data is not stored locally.     
+</Highlight>
+
+
+
+<Procedure>
+
+### Querying tiered data in a single query
+
+
+1. Enable `timescaledb.enable_tiered_reads` before querying the hypertable with tiered data and reset it after it is complete:
+
+```sql
+set timescaledb.enable_tiered_reads = true; SELECT count(*) FROM example; set timescaledb.enable_tiered_reads = false;
+```
+
+This queries data from all chunks including tiered chunks and non tiered chunks: 
+
+   ```sql
+   ||count|
+   |---|
+   |1000|
+   ```
+
+</Procedure>
+
+
+
+### Querying tiered data for an entire session
+
+All future queries within a session can be enabled to use data tiering by enabling `timescaledb.enable_tiered_reads` within a session. 
+
+<Procedure>
+
+1. Enable `timescaledb.enable_tiered_reads` for an entire session:
+
+```sql
+set timescaledb.enable_tiered_reads to true;
+```
+
+1. All future queries in that session are configured to read from tiered data and locally stored data.    
+
+</Procedure>
+
+
+### Querying tiered data in all future sessions
+
+You can also enable queries to read from tiered data always by following these steps:
+
+<Procedure>
+
+1. Enable `timescaledb.enable_tiered_reads` for all future sessions:
+
+```sql
+alter database tsdb set timescaledb.enable_tiered_reads to true;
```

Comment:
Need to verify this and correct, if needed, either in this PR or as part of a follow up PR.